app/views/govuk_publishing_components/components/docs/machine_readable_metadata.yml

```diff
@@ -0,0 +1,33 @@
+name: Machine readable metadata
+description: Generates schema.org JSON-LD for use by search engines
+body: |
+  This component doesn't output anything visible. It generates "structured data"
+  in JSON-LD format, with [schema.org](http://schema.org) schemas.
+accessibility_criteria: |
+  The HTML should not be visible.
+display_html: true
```

Comment:
This will start to work after https://github.com/alphagov/govuk_publishing_components/pull/320 is merged.